app/src/main/java/com/keylesspalace/tusky/util/NotificationHelper.java

```diff
@@ -569,22 +588,43 @@ private static String titleForType(Context context, Notification notification, B
             case REBLOG:
                 return String.format(context.getString(R.string.notification_reblog_format),
                         accountName);
+            case POLL:
+                if(notification.getStatus().getAccount().getId().equals(account.getAccountId())) {
```

Comment:
I don't even know if we need this distinction or generic "the poll has ended" would be enough but if it's there cool then